diff -p -up rezound-0.12.3beta/src/backend/Remaster/CAdaptiveNormalizeAction.cpp.gcc44 rezound-0.12.3beta/src/backend/Remaster/CAdaptiveNormalizeAction.cpp
--- rezound-0.12.3beta/src/backend/Remaster/CAdaptiveNormalizeAction.cpp.gcc44	2005-02-15 03:54:44.000000000 +0100
+++ rezound-0.12.3beta/src/backend/Remaster/CAdaptiveNormalizeAction.cpp	2009-06-02 04:19:58.000000000 +0200
@@ -27,6 +27,8 @@
 
 #include "../DSP/LevelDetector.h"
 
+#include <cstdio>
+
 //??? could use a state machine or some sort of fuzzy state machine that doesnt raise the gain unless there is going to be a rise again in some minimum amount of time (actually could probably just use a delay line to do this)
 
 
