--- rezound-0.12.2beta/src/frontend_fox/CMainWindow.cpp~	2005-09-05 00:31:10.000000000 +0200
+++ rezound-0.12.2beta/src/frontend_fox/CMainWindow.cpp	2006-02-14 14:28:19.000000000 +0100
@@ -433,7 +433,7 @@
 
 long CMainWindow::onSoundListChange(FXObject *sender,FXSelector sel,void *ptr)
 {
-	FXint index=(FXint)ptr;
+	long index=(long)ptr;
 
 	if(index>=0 && index<soundList->getNumItems())
 	{
@@ -1836,13 +1836,13 @@
 
 	if(pos==0 && FXSELID(sel)==ID_SHUTTLE_BACKWARD)
 	{
-		if((unsigned)ptr > 1/*came from keyboard event*/) enableAutoRepeat(getApp()->getDisplay(),false); // QQQ
+		if((unsigned long)ptr > 1/*came from keyboard event*/) enableAutoRepeat(getApp()->getDisplay(),false); // QQQ
 		shuttleDial->setValue(pos-inc);
 		onShuttleChange(sender,sel,ptr);
 	}
 	else if(pos==0 && FXSELID(sel)==ID_SHUTTLE_FORWARD)
 	{
-		if((unsigned)ptr > 1/*came from keyboard event*/) enableAutoRepeat(getApp()->getDisplay(),false); // QQQ
+		if((unsigned long)ptr > 1/*came from keyboard event*/) enableAutoRepeat(getApp()->getDisplay(),false); // QQQ
 		shuttleDial->setValue(pos+inc);
 		onShuttleChange(sender,sel,ptr);
 	}
--- rezound-0.12.2beta/src/frontend_fox/CPasteChannelsDialog.cpp~	2005-02-09 09:17:36.000000000 +0100
+++ rezound-0.12.2beta/src/frontend_fox/CPasteChannelsDialog.cpp	2006-02-14 14:30:22.000000000 +0100
@@ -206,7 +206,7 @@
 	{
 		pasteChannels.clear();
 
-		actionParameters->setValue<unsigned>(_("MixMethod"),(unsigned)(mixTypeComboBox->getItemData(mixTypeComboBox->getCurrentItem())));
+		actionParameters->setValue<unsigned long>(_("MixMethod"),(unsigned long)(mixTypeComboBox->getItemData(mixTypeComboBox->getCurrentItem())));
 		
 		if(repeatTypeComboBox->getCurrentItem()==0)
 		{ // repeating it a given number of times
--- rezound-0.12.2beta/src/frontend_fox/COggDialog.cpp~	2004-02-11 06:11:17.000000000 +0100
+++ rezound-0.12.2beta/src/frontend_fox/COggDialog.cpp	2006-02-14 14:30:53.000000000 +0100
@@ -136,7 +136,7 @@
 
 long COggDialog::onRadioButton(FXObject *sender,FXSelector sel,void *ptr)
 {
-	if((int)ptr==0) // only act when ptr==1 when it's getting checked
+	if((long)ptr==0) // only act when ptr==1 when it's getting checked
 		return 1;
 
 	// turn off all buttons
--- rezound-0.12.2beta/src/frontend_fox/CMp3Dialog.cpp~	2004-06-19 05:53:18.000000000 +0200
+++ rezound-0.12.2beta/src/frontend_fox/CMp3Dialog.cpp	2006-02-14 14:31:45.000000000 +0100
@@ -170,7 +170,7 @@
 
 long CMp3Dialog::onRadioButton(FXObject *sender,FXSelector sel,void *ptr)
 {
-	if((int)ptr==0) // only act when ptr==1 when it's getting checked
+	if((long)ptr==0) // only act when ptr==1 when it's getting checked
 		return 1;
 
 	// turn off all buttons
